 Traceback (most recent call last):
 File "<pyshell#37>", line 1, in <module>
   clf = clf.fit(CS_tr,SN_tr.toarray())
 File "C:\Users\Ekkasit Smithipanon\AppData\Local\Programs\Python\Python36- 
32\lib\site-packages\scipy\sparse\compressed.py", line 943, in toarray
   out = self._process_toarray_args(order, out)
 File "C:\Users\Ekkasit Smithipanon\AppData\Local\Programs\Python\Python36- 
32\lib\site-packages\scipy\sparse\base.py", line 1130, in 
_process_toarray_args
   return np.zeros(self.shape, dtype=self.dtype, order=order)
ValueError: array is too big; `arr.size * arr.dtype.itemsize` is larger than 
the maximum possible size.
